.github/workflows/ci_windows.yml

```diff
@@ -0,0 +1,52 @@
+name: Run Windows CI
+
+on: [push, pull_request]
+
+env:
+  DEPS_BUILD_DIR: "C:/projects/nvim-deps"
+  DEPS_PREFIX: "C:/projects/nvim-deps/usr"
+  # Silence/redirect errors due to missing locking support (via libgcov).
+  # GCOV_ERROR_FILE: "$(TEMP)/libgcov-errors.log"
+
+jobs:
+  test:
+    runs-on: windows-2016
+    strategy:
+      matrix:
+        config: [ MINGW_64-gcov, MINGW_32, MSVC_64, MSVC_32 ]
+    name: Build and test
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        with:
+          path: C:\projects\nvim-deps
+          key: ${{ matrix.config }}-${{ hashFiles('third-party\**') }}
+
+      - uses: actions/setup-python@v2
```

Comment:
I'm not sure we actually need to use these actions.  Their intent, as I understand it, is to ensure the specified Python version is provided in `%PATH%`.  Is the second one actually additive or does it override the first?

`build.ps1` was also updated to use the full paths to the existing tools, so I think that's also an argument for these being unnecessary.

Reply:
I tried it and these were not necessary. It seems to be just a matter of adding the path to the beginning of  `%PATH%`.